<launch>

  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <rosparam file="$(find bitathome_navigation_control)/config/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find bitathome_navigation_control)/config/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find bitathome_navigation_control)/config/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find bitathome_navigation_control)/config/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find bitathome_navigation_control)/config/base_local_planner_params.yaml" command="load" />
  </node>

</launch>
